knimin/lib/tests/test_parse.py

```diff
@@ -97,6 +101,115 @@ def test_parse_qpcr_object(self):
             "u'Status']")
 
 
+class EchoParserTests(TestCase):
+    def setUp(self):
+        base = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'data')
```

Comment:
I wonder if it's easier to do `from os.path import join, dirname, realpath` then do `join` directly, like other scripts in the same project did.